action.sh

```diff
@@ -65,6 +80,7 @@ generate () {
 
             docker run -d \
                 -v $ACTION_PATH/containers/postgres/initdb.sh:/docker-entrypoint-initdb.d/initdb.sh \
+                -v $ROLES_FILE_PATH:/docker-entrypoint-initdb.d/init_roles.sh \
```

Comment:
mounting under this directory will cause the postgres server to execute the script after startup